app/models/teleconsultation.rb

```diff
@@ -0,0 +1,42 @@
+class Teleconsultation < ApplicationRecord
+  include Mergeable
+
+  belongs_to :patient
+  belongs_to :medical_officer, class_name: "User", foreign_key: :medical_officer_id
+  belongs_to :facility, optional: true
```

Comment:
Is this optional because the `record` may arrive first?

Reply:
Yes.